django/contrib/sessions/backends/base.py

```diff
@@ -20,6 +21,13 @@
 # on case insensitive file systems.
 VALID_KEY_CHARS = string.ascii_lowercase + string.digits
 
+# the delimiter cannot be a member of VALID_KEY_CHARS or a file directory
+# component. It can only be a single character.
+SESSION_KEY_DELIMITER = '$'
+# this should be available in hashlib
+SESSION_HASHING_ALGORITHM = 'sha256'
+SESSION_HASHED_KEY_PREFIX = SESSION_HASHING_ALGORITHM + SESSION_KEY_DELIMITER
+
```

Comment:
Is there a reason to add these as module constants? I would rather move them to the `HashingSessionBase`. Also we don't need to allow customizing delimiter.